app/plugins/ConductrPlugin.scala

```diff
@@ -0,0 +1,20 @@
+package plugins
+
+import com.typesafe.conductr.bundlelib.play.StatusService
+import com.typesafe.conductr.bundlelib.play.ConnectionContext.Implicits
+import play.api.{ Application, Plugin, Logger }
+import play.api.libs.concurrent.Execution
+
+class ConductrPlugin extends Plugin {
+  
+  override def onStart() {
+    import Implicits.defaultContext
+    StatusService.signalStartedOrExit()
+    Logger.info("Application has started")
```

Comment:
The question of whether to make loggers injectable or not has long been hotly debated, like whether they should be static or instance fields.

My personal preference and the coding standard everywhere where I've worked has been for them to be static, since loggers are looked up by class, making a logger associated with a class, not an instance.  This is primarily how most current logging libraries are designed to be used, eg slf4j, log4j, etc.  Another argument for this is that it's not always known up front when you might want to log some debug out, so you may implement something as a pure function, in an object, wherever, but then in future you need to put some debug statement in there, and you can't do that without breaking your API.  Sometimes, you want to log something about some behaviour being deprecated, but if you have to add a logger parameter to your API, then you've lost all advantages of deprecation in the first place because you've broken everything anyway, and then when you remove the deprecation log statement, you'll remove the log parameter, breaking the API again... this is why it just feels like logging shouldn't be an injectable dependency but rather should be static, things get very messy to achieve tasks that should be very simple, like aiding debugging, or warning the user of something, etc.

If we were to implement this, I don't think it makes sense to make `Logger` injectable, as that means the whole application uses the same logger, which makes it impossible to control logging for individual parts of an application.  It would make more sense to make `LoggerFactory` injectable.  I've had a look, I can't see anything in Guice that would let us automatically inject a logger configured for a particular class into that class.